tests/php/Control/RequestHandlingTest.php

```diff
@@ -25,16 +24,6 @@ class RequestHandlingTest extends FunctionalTest
     protected static $fixture_file = null;
 
     protected static $illegal_extensions = array(
-        // Suppress CMS error page handling
-        Controller::class => array(
-            ErrorPageControllerExtension::class,
-        ),
-        Form::class => array(
```

Comment:
Can do. I will hardcode a string rather than using `::class` as we'd want to avoid a tight coupling to the ErrorPage module, right?